src/modules/Extension/Controller/Admin.php

```diff
@@ -77,6 +77,9 @@ public function get_langs(\Box_App $app)
     public function get_settings(\Box_App $app, $mod)
     {
         $this->di['is_admin_logged'];
+        $extensionService = $this->di['mod_service']('Extension');
+        $extensionService->checkPermission($mod, $app);
```

Comment:
It's not a big thing but, in get_settings, the parameters are $app, $mod and in checkPermission it's ($mod, $app).

Would be more intuitive to have both the same way around.

Reply:
I get that, but in this case the order was chosen because the `$mod` param is required for the function and the `$app` param is not. Having the optional parameter first while still technically possible (by passing `null` to it) is also un-intuitive, more-so than the parameter order differing from the function that just happens to be calling it (imo)